src/engine.rs

```diff
@@ -1582,8 +1840,15 @@ pub trait ProcessRun: Send + Sync {
 
 /// A type that can process the valid data from an RPKI publication point.
 pub trait ProcessCa: Sized + Send + Sync {
+    /// Sets the index of repository in the processing run metrics.
+    fn repository_index(&mut self, repository_index: usize) {
+        let _ = repository_index;
```

Comment:
Not sure what this is supposed to do, is this intentional? Same goes for `fn update_refresh` below.

Reply:
This prevents an unused variable warning. Normally you would prefix the variable name with an underscore, but since this is an argument and it shows up in the documentation, I didn’t want to do that.